### PR TITLE
Creating user_proxy_path route, spec test,  and #show_proxies action

### LINKED ISSUE #362: Remove person requirement for proxy depositors
Instead of relying on retrieving the person on each page, simply to generate a URL to the "My Proxies" page, instead have the link be a user_proxy_path; This would be analogous to the "My Profile" link (https://github.com/projecthydra/curate/blob/master/lib/curate/rails/routes.rb#L18) and could be a new action on the UserProfileController

https://github.com/projecthydra/curate/blob/master/app/views/shared/_my_actions.html.erb#L10

### PR DESCRIPTION
**Pull request by scherztc**
Fixes #362

### CODE REVIEW

File: lib/curate/rails/routes.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ def curate_for opts=nil
           resources :depositors, only: [:index, :create, :destroy]
         end
         match 'profile' => 'user_profiles#show', via: :get, as: 'user_profile'
+        match 'user/proxies' => 'user_profiles#show_proxies', via: :get, as: 'user_proxy'
```
Comment (jcoyne) [inline lib/curate/rails/routes.rb]: Why are you giving this an "as:" clause? It doesn't appear to be used or tested.  Furthermore, it should be a plural form. and written as: `get 'user/proxies' => 'user_profiles#show_proxies'`

**Comment by jcoyne** [comment]
Wasn't issue #362 already resolved by 0c0ec0c2b99fc6966fb905c94d2216989d7b59c1 ?

File: lib/curate/rails/routes.rb (hunk 1 above)
Comment (scherztc) [inline lib/curate/rails/routes.rb]: Noted.  Thanks.

**Comment by scherztc** [comment]
Yes, it was.  Jeremy completed it with his Rails 4.0.2 refactoring.  He still wanted me to complete my issue as an experience exercise.

**Comment by jcoyne** [comment]
We should either change the view to use this new route, or we should close this PR.  I don't want to have dead code in the gem.  @jeremyf what do you want to do?

**Comment by scherztc** [comment]
I initially changed the link_to in app/views/shared/_my_actions.html.erb but had a merge conflict with Jeremy's changes so I left his route.

**Comment by jeremyf** [comment]
@jcoyne I'm going to review the code, but the plan was to close the pull request without merge.

**Comment by jeremyf** [comment]
This was a good test exercise, but I'm going to close the pull request.

**Pull request closed without merging**